.github/workflows/ci.yml

```
@@ -32,7 +32,7 @@ jobs:
         run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev
         if: runner.os == 'linux'
       - name: Build & run tests for all feature combinations
-        run: cargo hack test --feature-powerset --group-features "2d","3d"
+        run: cargo hack test --feature-powerset --group-features "2d","3d" --group-features "stageless","progress_tracking_stageless"
```

```suggestion
        run: cargo hack test --feature-powerset --group-features "2d","3d","dynamic_assets" --group-features "stageless","progress_tracking_stageless"
```
This should fix the CI.